### PR TITLE
[DOCS] Update runbook-vet-center-new.md

### PR DESCRIPTION
**Pull request by davidconlon**
Lots of edits. Please read carefully and make sure to add issues for outstanding email templates/kb articles as necessary.

## Description

Relates to #_issueid_. (or closes?)

## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user _uid_ with _user_role_
1. Do this
   - [ ] Validate that
2. Then
   - [ ] Validate that
3. Then validate Acceptance Criteria from issue
   - [ ] a
   - [ ] b
   - [ ] c

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [ ] `Facilities`
- [ ] `User support`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Review by swirtSJW — DISMISSED** [review]
Changes look good and make sense.

File: .github/ISSUE_TEMPLATE/runbook-vet-center-new.md [hunk 1]
```diff
@@ -36,31 +36,30 @@ Facility API ID: <insert_facility_API_ID>
...
+
+**If a Outstation:**
+- [ ] Confirm what Vet Center it belongs to, and set the "Main Vet Center location" field. The parent location may be derived by the Facility ID.
+- [ ] Contact the Vet Center editor to remind them to (1) add a photo of the Outstation and then they can publish when ready
+- [ ] Consider: Following up with Barb/RCS Central office to be sure that she's aware that the Outstation is in-flight
 
 **If a Vet Center:**
 - [ ] 1. Become aware that the new facility is now on the Facility API (typically, via a Flag).
-- [ ] 2. Check with Readjustment Counseling Services to confirm what district the Vet Center belongs to. 
+- [ ] 2. Check with Readjustment Counseling Services to (1) confirm what district the Vet Center belongs and (2) Identify the Vet Center Director and Outreach Specialist (names/email addresses)
```
Comment (jilladams) [inline .github/ISSUE_TEMPLATE/runbook-vet-center-new.md:50]: @mmiddaugh do you know: once we've identified the Director/Outreach Specialist, does that info just live in ticket for final steps related to the runbook? Or should that data make its way anywhere else (a directory somewhere that we use for something, etc)?

File: .github/ISSUE_TEMPLATE/runbook-vet-center-new.md [hunk 2]
```diff
@@ -36,31 +36,30 @@ Facility API ID: <insert_facility_API_ID>
...
 - [ ] 3. In [Sections taxonomy](https://prod.cms.va.gov/admin/structure/taxonomy/manage/administration/overview), move the Vet Center Section to the appropriate district.
-- [ ] 4. Communicate with editor (do they need to be onboarded)
-
-[@TODO write sample email - SEE runbook-vamc-facility-new]
-
-- [ ] 5. Create a [URL change request](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/new?assignees=&template=runbook-facility-url-change.md&title=URL+Change+for%3A+%3Cinsert+facility+name%3E). (**Note: The URL change request ticket blocks the completion of this ticket.**)
-
-<insert_url_change_request_link>
```
Comment (jilladams) [inline .github/ISSUE_TEMPLATE/runbook-vet-center-new.md:51]: Redirects don't apply?

Comment (swirtSJW) [reply .github/ISSUE_TEMPLATE/runbook-vet-center-new.md:51]: For a new facility there should be no redirects needed.  The only update is adding the new url to the CSV that lighthouse ingests.  That is true up to the point where they start using the CMS as the source of truth for the url.

File: .github/ISSUE_TEMPLATE/runbook-vet-center-new.md (hunk 1 above)
Comment (swirtSJW) [reply .github/ISSUE_TEMPLATE/runbook-vet-center-new.md:50]: Once they are assigned to the new vet center, then those people will have accounts in the CMS and the CMS will be the record of who is the Director/outreach specialist.

Comment (jilladams) [reply .github/ISSUE_TEMPLATE/runbook-vet-center-new.md:50]: This is for the purposes of the ticket / access credentials, does not need to be centrally documented ongoing.

**Review by omahane — APPROVED** [review]
I made a couple slight changes to make sure the flag name and style was consistent. This looks right to me, otherwise.

**Review by swirtSJW — APPROVED** [review]

**Pull request merged**